code/datums/traits/good.dm

```diff
@@ -0,0 +1,58 @@
+//predominantly positive traits
+//this file is named weirdly so that positive traits are listed above negative ones
```

Comment:
No.

Reply:
What part is no? I named it `good.dm` so it alphabetized over `negative.dm`.

Reply:
Anything relying on compile time for in-game ordering shouldn't exist. Unless you just mean in some editor view.

Reply:
It's for the list to go
`positive`
`negative`
`neutral`
It has no intentional impact on the way things look in the actual panel.

Reply:
You're still relying on compilation order here, use some explicit priority. (Sorting by value being the obvious one)